packages/core/src/errors/RequestError/request-error.test.ts

```diff
@@ -21,7 +21,7 @@ describe('RequestError', () => {
     expect(newRequestError.code).toEqual(errorCode);
     expect(newRequestError.expose).toEqual(true);
     expect(newRequestError.message).toEqual(expectMessage);
-    expect(newRequestError.body).toEqual({
+    expect(newRequestError.toBody(i18next as unknown as i18n)).toEqual({
```

Comment:
are these two `as` required?

Reply:
removed. Import i18next from utils instead. It has the proper type definition.